### PR TITLE
Move src/Schedulers/* to src-internal/Admin/Schedulers

### LINKED ISSUE #8276: Make `src/Schedulers/*` internal
Except for `SchedulerTraits.php` - that can be left public.

Move to `src/Schedulers/Internal/` and decorate public functions with `@internal` comment

5 files

My estimate: 4 hours

### PR DESCRIPTION
**Pull request by moon0326**
Fixes #8276 

This PR moves `src/Schedulers/*` to `src-internal/Admin/Scheulders`

### Detailed test instructions:

**CustomersScheduler.php**

1. Navigate to `WooCommerce -> Settings-> Accounts & Privacy`
2. Check on `Allow customers to create an account during checkout` and save changes.
3. Add a new simple product.
3. Open an incognito browser session and navigate to `http://your-site/shop`
4. Click the product you just added and continue to the checkout page.
5. On the checkout page, click on"Create account?" at the bottom of the page
6. Complet the checkout.
7. Navigate to `WooCommerce -> Orders` and change the status of the order to complete.
8. Navigate to `Analytics -> Orders` and confirm the order you just placed.
9. Navigate to `WooCommerce -> Customers` and click the user account you created during the checkout.
10. Change last and first name.
11. Navigate to `WooCommerce -> Status -> Scheduled Actions` and confirm `wc-admin_import_customers` hook has been created.

Extra: Navigate back to `Analytics -> Orders` and confirm the customer's last and first name has been updated. It might take some time to see the updated info. 

**OrderScheduler.php**

1. Add a new simple product.
2. Create a new order and mark it as complete.
3. Navigate to `WooCommerce -> Orders` and click the order.
4. Click `Refund` button and refund the order total.
5. Navigate to `WooCommerce -> Status -> Scheduled Actions` and confirm a new `wc-admin_import_customers` has been created.

**MailchimpScheduler.php**

1. Start with a fresh install.
2. Start OBW.
3. Make sure to check `Get tips, product updates and inspiration straight to your mailbox`.
4. Install and activate [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/) plugin
5. Navigate to `Tools -> Cron Events`
6. Run `wc_admin_daily`
7. Navigate to `WooCommerce -> Status -> Logs`.
8. Select `mailchimp_scheduler-2022-MM-DD-:hash.log` file. MM-DD should be the current date. 
9. You should see `ERROR Incorrect response from Mailchimp API with: ` error message if you're not running local woocommerce.com. The error confirms that the scheduler is sending a request.

no changelog

### CODE REVIEW

File: src-internal/Admin/Schedulers/CustomersScheduler.php [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
-namespace Automattic\WooCommerce\Admin\Schedulers;
+namespace Automattic\WooCommerce\Internal\Admin\Schedulers;
 
 defined( 'ABSPATH' ) || exit;
 
 use \Automattic\WooCommerce\Admin\API\Reports\Cache as ReportsCache;
 use \Automattic\WooCommerce\Admin\API\Reports\Customers\DataStore as CustomersDataStore;
-use \Automattic\WooCommerce\Admin\Schedulers\OrdersScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\ImportScheduler;
+use Automattic\WooCommerce\Internal\Admin\Schedulers\OrdersScheduler;
```
Comment (chihsuan) [inline src-internal/Admin/Schedulers/CustomersScheduler.php:13]: @moon0326 I just found my PhpStorm shows lines 12-13 are unnecessary, though VSCode doesn't show this. I saw you also add additional imports to `OrdersScheduler.php` and `ImportScheduler.php`. Do we prefer these explicit imports?

Screen Shot 2022-02-16 at 11 46 25

File: src-internal/Admin/Schedulers/CustomersScheduler.php [hunk 2]
```diff
@@ -3,13 +3,16 @@
...
+use Automattic\WooCommerce\Admin\Schedulers\WC_Order;
+use Automattic\WooCommerce\Admin\Schedulers\WP_User_Query;
```
Comment (chihsuan) [inline src-internal/Admin/Schedulers/CustomersScheduler.php:15]: I might be lack understanding of this. I couldn't find the definition files for `Automattic\WooCommerce\Admin\Schedulers\WC_Order` and `Automattic\WooCommerce\Admin\Schedulers\WP_User_Query;`. Have we defined them somewhere?

**Review by chihsuan — COMMENTED** [review]
@moon0326 Thanks for providing detailed test instructions. Overall looks good and tests well! I just wanted to confirm two "imports" issues before approving this.

File: src-internal/Admin/Schedulers/CustomersScheduler.php (hunk 1 above)
Comment (moon0326) [reply src-internal/Admin/Schedulers/CustomersScheduler.php:13]: Thank you for spotting them. They should be removed. I'm not sure why my PHPStorm added them (or did not remove them). I'll make an update.

File: src-internal/Admin/Schedulers/CustomersScheduler.php (hunk 2 above)
Comment (moon0326) [reply src-internal/Admin/Schedulers/CustomersScheduler.php]: Fixed in https://github.com/woocommerce/woocommerce-admin/pull/8316/commits/8d72e683f91470c16ef43fd47357051893983af2

File: src-internal/Admin/Schedulers/CustomersScheduler.php (hunk 1 above)
Comment (moon0326) [reply src-internal/Admin/Schedulers/CustomersScheduler.php]: Fixed in https://github.com/woocommerce/woocommerce-admin/pull/8316/commits/149bd59853ba54715bbb8e796a2963e906e473dc

**Review by chihsuan — APPROVED** [review]
Looks good! Thanks for removing the unnecessary imports. 🚢

**Pull request merged**